Ally scripts initial ideas (#1)
sarahsemicolon/scripts-for-gender-freedom: talking-about-their-pronouns.md

# Asking someone's pronouns

## Over Slack / in conversation

> What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?

Author: Is this a good way to ask?

Reviewer: ```suggestion
When: In introducing oneself to someone new
What: Hi, I'm Janet Jackson, and I go by `they/them` pronouns. How should I refer to you?

When: For someone you already know
What: By the way, I use `he/him` pronouns. How should I refer to you?
```
Adapted from https://pronouns.org/asking. You may be asking someone who doesn't know about pronouns, so by offering an example of the expected answer, it might help the prompt.

I also have a personal bugbear about saying "sorry" when I've not done anything wrong! :sweat_smile:

# Correcting someone who misgenders someone else

> 

Reviewer: ```suggestion
:warning: If the person is in the conversation, they may be happy to correct it themselves, or leave it be and handle it via DMs, so exercise caution in doing this if they are present. They are, after all, the authority on how they wish to be addressed!

:bulb: YMMV, but I'd only correct if I was sure I had a current source of pronouns the person would like to use - Slack, Workday, a direct personal conversation. If you're not sure 100% yourself, I'd strongly advise not doing this. 
```

Author: Yep, I always go and check Slack or something unless we've explicitly had that conversation before

## During a meeting

### Respond in the moment

When: Once the person who's just got the pronouns incorrect has finished talking
How: Verbally

> What: Just to point out, I'm pretty sure their pronouns are they/them

Reviewer: ```suggestion
What: [when you next refer to them, use the pronouns you know the person uses]
```

I think this is a really cool idea - it certainly reduces the cognitive load on me if someone else is able to do this for me, especially when I'm not around.

I would suggest not making a big deal of this in the moment - you're likely gathered for some other reason apart from their pronouns!

Author: I like to go beyond just using the right pronouns as it feels almost passive aggressive to just use the right one without explaining why? Maybe this should highlight the context dependency - sometimes I'd definitely just mention it later but sometimes it can just be a quick thing before the conversation moves on? I like doing it explicitly if I'm in a conversation where no one present knows the person's gender too to challenge assumptions of gender (most commonly from experience: they're a dev so they're male)

### Respond async

Where: Slack DM, email etc.

> What: FYI, I think you used "she" to refer to Sarah and I'm pretty sure their pronouns are they/them :)

Reviewer: ```suggestion
When: As soon as it happens
What: FYI, Kieran's pronouns are they/them - I saw it in their Slack profile / they told me last week/month/when we met :)
```

a) I'd prefer we didn't embed the wrong pronouns by mentioning them! :sweat_smile: 
b) It gives them a tool for getting things right for that person, and others, in future.

Author: a) My bad
b) Good point

I was trying to avoid the same sort of thing as earlier where you give context because the person might not explicitly think about pronouns.

Maybe we can use your suggestion earlier with
`FYI, Kieran prefers to be referred to with they or them - I saw it in their Slack profile / they told me last week/month/when we met :)` ?

## Via Slack/email

Where: In a direct message to that person (Slack DM, email, i.e. not in the initial thread itself to avoid derailing conversation if it happened in a group conversation)
What: FYI, I'm pretty sure Sarah uses they/them pronouns :)

# When you get pronouns wrong

## If you realise during the interaction

> When: In speech, not as a big deal

Author: Is this the best way? Is it worth a comment or just fixing it? Does this all depend on the person?

Reviewer: All of this is very depends-on-the-person - in my comments/suggestions I've just been going with what I personally prefer!

Again, this may not be everyone's preferred route, but I prefer folks not necessarily apologising for getting it wrong, and just simply getting it right in future - similar to if you were pronouncing a name wrong. In the moment you might say "Ben, sorry, Benjamin" to someone who prefers their full name used, and just move on, rather than saying "I'm sorry I keep calling you Ben! It's really difficult / I'll keep trying / etc". I'm never really quite sure what to say in response to it, in all honesty!

Author: Maybe that's a good example to add in? People are likely to be better at that (/ I guess maybe find it less awkward?) so it might be a helpful mental model?

```
What: She, sorry, they

:bulb: In the same way that if you accidentally called someone the wrong name you'd quickly correct rather than making a big deal (e.g. `Nick, sorry, Rick`) you can correct yourself on pronouns too!

How: Verbally
What: Sorry, I just realised I've been getting your pronouns wrong.

## If you realise later

Where: In a direct message to that person (Slack DM, email etc.)

> What: Hey, I realised / got told I got your pronouns wrong earlier, sorry. I'm try and sometimes I mess up but I know it's important and I'll keep working at it. Feel free to point out if it seems I've forgotten.

Author: This might be far too long-winded?

Reviewer: Similar to my above comment: if it was in the moment, I'd obviously apologise for getting it wrong - "they, sorry, he" - but I wouldn't dwell on it (unless I'd particularly fouled up, in which case we're not in script-territory any more, and you will have to exercise Full Personal Relationship skills!)

Author: I think some people would want to convey that they've noticed and are sorry, though that might be for their benefit rather than the person they've misgendered...

Maybe this needs qualifying with "It's probably enough just to get it right next time but you could always drop them a message along the lines of ..."?
